### PR TITLE
Removed the linter - misspel 

### LINKED ISSUE #2962: Enable linters for test packages
**Is your feature request related to a problem? Please describe.**
As of now linters are disabled from test packages in .golangci.yaml file:
```
issues:
  exclude-rules:
    - path: '_test.go'
      linters:
        - errcheck                      # Errors may be ignored in tests.
        - unparam                       # Tests might have unused function parameters.
        - lll
        - dupl
        - misspell
        - nestif
        - gci
```
**Describe the solution you'd like**
After enabling this linter, fix all kind of linter issues coming up in test packages.

### PR DESCRIPTION
**Pull request by anishbista60**
## Change Overview

- Removed the  linter `misspell` 
## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test
- [ ] :building_construction: Build

## Issues 

- fixes #2962 

## Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Comment by anishbista60** [comment]
@julio-lopez @saima-s  All changes looks fine here , as i test in my local .
 Except below three.  

```
pkg/secrets/repositoryserver/aws_secrets_test.go:19: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/kanisterio/kanister) -s blank -s dot (gci)

pkg/secrets/repositoryserver/azure_secrets_test.go:19: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/kanisterio/kanister) -s blank -s dot (gci)

pkg/secrets/repositoryserver/gcp_secrets_test.go:19: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/kanisterio/kanister) -s blank -s dot (gci)
```
what is missing here?  i have arranged these package based on 
[https://github.com/kanisterio/kanister/blob/adab42bfd592f034ece0a3b9595218b87eb84d0f/.golangci.yml#L77](url)

File: .golangci.yml [hunk 1]
```diff
@@ -44,9 +44,7 @@ issues:
         - unparam                       # Tests might have unused function parameters.
         - lll
         - dupl
-        - misspell
         - nestif
-        - gci
```
[the remark this answers is not in the document]
Comment (julio-lopez) [reply .golangci.yml:49]: ☝🏼

**Review by julio-lopez — CHANGES_REQUESTED** [review]

File: .golangci.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (anishbista60) [reply .golangci.yml:49]: @julio-lopez Done sir .

File: pkg/secrets/repositoryserver/azure_secrets_test.go
Comment (julio-lopez) [inline pkg/secrets/repositoryserver/azure_secrets_test.go:1]: This one should be reverted. The import order was correct before the change.

File: pkg/secrets/repositoryserver/aws_secrets_test.go [hunk 2]
```diff
@@ -16,11 +16,13 @@ package repositoryserver
...
 import (
 	"github.com/pkg/errors"
-	. "gopkg.in/check.v1"
+
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	secerrors "github.com/kanisterio/kanister/pkg/secrets/errors"
+
+	. "gopkg.in/check.v1"
```
Comment (julio-lopez) [inline pkg/secrets/repositoryserver/aws_secrets_test.go:25]: This one should be reverted. The import order was correct before the change.

```suggestion
	. "gopkg.in/check.v1"
	corev1 "k8s.io/api/core/v1"
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"

	secerrors "github.com/kanisterio/kanister/pkg/secrets/errors"
```

File: pkg/secrets/repositoryserver/gcp_secrets_test.go
Comment (julio-lopez) [inline pkg/secrets/repositoryserver/gcp_secrets_test.go:1]: Revert this one.

File: pkg/secrets/repositoryserver/s3compliant_secrets_test.go
Comment (julio-lopez) [inline pkg/secrets/repositoryserver/s3compliant_secrets_test.go:1]: and this one

**Review by julio-lopez — CHANGES_REQUESTED** [review]
Actually, the import changes need to be reverted as well.
The original ordering was correct.

It seems that the `.` import is throwing off `gci`.

**Comment by anishbista60** [comment]
@julio-lopez i think i  should have next PR for `gci`.

**Review by julio-lopez — APPROVED** [review]

**Pull request merged**